app/konnect/reference/plugins.md

```diff
@@ -13,6 +13,9 @@ and lets you create _dynamic_ dependencies between plugins.
 
 To configure this setting in {{site.konnect_short_name}}, go to **Gateway Manager > Plugins**, and then select **Configure Dynamic Ordering** from the context menu next to the plugin you want to configure. From the plugin ordering settings, you can configure whether a plugin runs before or after another plugin.
 
+{:.note}
+> **Note**: There are limitations to dynamic plugin ordering. Particularly, dynamic plugin ordering can't coexist with consumer-scoped plugins, even if they are applied to entirely different service and route pairs, or are running globally. For more information, see [Known Limitations in Dynamic Plugin Ordering](/gateway/{{page.release}}/kong-enterprise/plugin-ordering/#known-limitations).
```

Comment:
```suggestion
> **Note**: There are limitations to dynamic plugin ordering. Particularly, dynamic plugin ordering can't coexist with consumer-scoped plugins, even if they are applied to entirely different service and route pairs, or are running globally. For more information, see [Known Limitations in Dynamic Plugin Ordering](/gateway/latest/kong-enterprise/plugin-ordering/#known-limitations).
```
Setting the link to `latest`, as the page it's on doesn't have a version so it doesn't generate one.